splash/tests/lua_modules/emulation.lua

```diff
@@ -26,7 +26,7 @@ function Splash:go_and_wait(args)
     error("non-zero 'wait' is required when rendering whole page")
   end
 
-  self:set_images_enabled(self.args.images)
+  self.images_enabled = self.args.images
```

Comment:
This was the only place set_images_enabled function is tested; if you're removing it another testshould be aded to ensure the function remains working.